scenarios.md

```diff
@@ -194,6 +194,42 @@ A deployment requires a mydb image. The mydb image is routinely updated for secu
 1. The release management system deploys the new `mydb:1.0-202002131000` image.
 1. The production orchestrator validates it's signed with the Acme Rockets production key.
 
+
+
+### Scenario #7: Malicious and Erroneous Actions
+
+#### Scenario 7.1: A Repository Compromise Occurs
+An attacker manages to compromise a repository and gain access to the repository as an administrator.   
```

Comment:
I'm not really sure we should bring in repository management and authorization scenarios to this, especially as there is not a common permissions model among the existing registries that are available today.

Reply:
I'd be interested to chat more and see if there is a way to capture these models and see how they may change things in the case of a compromise.  Ideally our design would work well under all current and envisioned permissions models...